json_generator/server.py

```
@@ -0,0 +1,69 @@
+@app.post("/chat", response_model=ChatResponse)
+async def chat_endpoint(req: ChatRequest):
+    try:
+        res = await chat_manager.handle_message(req.session_id, req.message)
+        return ChatResponse(
+            session_id=req.session_id,
+            response=res["message"],
+            json_schema=res["json_schema"],
+        )
+    except Exception as e:
+        logging.error(e)
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)